packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1899,12 +1917,25 @@ contract OPContractsManager is ISemver {
 
     /// @notice Upgrades a set of chains to the latest implementation contracts
     /// @param _opChainConfigs Array of OpChain structs, one per chain to upgrade
-    /// @dev This function is intended to be called via DELEGATECALL from the Upgrade Controller Safe
+    /// @dev This function is intended to be called via DELEGATECALL from the Upgrade Controller Safe.
+    /// @dev This function assumes that each chain's superchainConfig is already upgraded.
```

Comment:
I think this is more accurate, since we'll check the status before doing the upgrade
```suggestion
    /// @dev This function requires that each chain's superchainConfig is already upgraded.
```